### PR TITLE
updated pacget's pacscript

### PR DESCRIPTION
**Pull request by erikLundstedt**
it now gets the version from git for better updates
i also moved everything to pacget-git despite that being the only version
and i updated the packagelist to reflect the changes

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
I don't thing you should remove the `-git` suffix, as people who download the `-git` version actually understand that they are installing the `-git` version (and the pros/cons with it, like more unstable/frequent updates). So I think removing the suffix is kind of hiding that fact.

---

> i also moved everything to pacget-git as despite that being the only version

There is a fetching software called [fastfetch](https://github.com/LinusDierheimer/fastfetch), which has it's AUR PKGBUILD script with the suffix `-git` even though it's their only way of distributing it to arch users.
*PS: I maintain their [pacscript version](https://github.com/pacstall/pacstall-programs/blob/master/packages/fastfetch-git/fastfetch-git.pacscript), and I also named it with the `-git` suffix*

**Comment by erikLundstedt** [comment]
> I don't thing you should remove the `-git` suffix, as people who download the `-git` version actually understand that they are installing the `-git` version (and the pros/cons with it, like more unstable/frequent updates). So I think removing the suffix is kind of hiding that fact.
> 
> > i also moved everything to pacget-git as despite that being the only version
> 
> There is a fetching software called [fastfetch](https://github.com/LinusDierheimer/fastfetch), which has it's AUR PKGBUILD script with the suffix `-git` even though it's their only way of distributing it to arch users.
> _PS: I maintain their [pacscript version](https://github.com/pacstall/pacstall-programs/blob/master/packages/fastfetch-git/fastfetch-git.pacscript), and I also named it with the `-git` suffix_

I added the `-git` suffix I didn't remove it

File: packages/pacget-git/pacget-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+name="pacget-git"
+pkgname="pacget"
+pkgdir="/usr/src/pacstall/${name}"
+
+maintainer="Erik Lundstedt <erik@lundstedt.it>"
+url="https://gitlab.com/Erik.Lundstedt/pacget.git"
+
+license="Don't Be a Jerk"
+depends="fzf curl bash"
+gives="$pkgname"
```
Comment (wizard-28) [inline packages/pacget-git/pacget-git.pacscript:12]:
```suggestion

```
Not needed!

File: packages/pacget-git/pacget-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#!/bin/env bash
+#-*- mode: sh ;mode:lsp;-*-
+name="pacget-git"
+pkgname="pacget"
+pkgdir="/usr/src/pacstall/${name}"
+
+maintainer="Erik Lundstedt <erik@lundstedt.it>"
+url="https://gitlab.com/Erik.Lundstedt/pacget.git"
+
+license="Don't Be a Jerk"
```
Comment (wizard-28) [inline packages/pacget-git/pacget-git.pacscript:10]:
```suggestion
license="Don't Be a Jerk"
```
Not needed!

File: packages/pacget-git/pacget-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+depends="fzf curl bash"
+gives="$pkgname"
+description="A fully foss, lightweight fuzzysearching TUI frontend for pacstall using fzf"
+removescript="yes"
+
+debian=TRUE
+ubuntu=TRUE
```
Comment (wizard-28) [inline packages/pacget-git/pacget-git.pacscript:17]:
```suggestion
debian=TRUE
ubuntu=TRUE
```
Not needed!

File: packages/pacget-git/pacget-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+#this just coppies the files over and makes the file executable
+install() {
+	sudo install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+    sudo install -Dm755 "pacget" "/usr/src/$name/pacget/usr/bin/pacget"
+
+}
+
+removescript() {
+	rm -rf /usr/src/pacstall/pacget
+}
```
Comment (wizard-28) [inline packages/pacget-git/pacget-git.pacscript:44]:
```suggestion
removescript() {
	sudo rm -rf "/usr/share/licenses/${pkgname}"
}
```

No need to remove `${pkgdir}` (it gets deleted automatically), just delete the license directory

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/pacget-git/pacget-git.pacscript (hunk 3 above)
Comment (erikLundstedt) [reply packages/pacget-git/pacget-git.pacscript:17]: i think its nice to have
although best case scenario would be if something similar got implemented directly into pacstall

Comment (wizard-28) [reply packages/pacget-git/pacget-git.pacscript:16]: okay.

File: packages/pacget-git/pacget-git.pacscript (hunk 2 above)
Comment (erikLundstedt) [reply packages/pacget-git/pacget-git.pacscript:10]: booth the program itself and the pacscript have that licence and i prefer putting the licence-notice on there even if its not needed

File: packages/pacget-git/pacget-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# pacget is a bashscript, no need to build anything
+build() {
+    true
+}
+#this just coppies the files over and makes the file executable
+install() {
+	sudo install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+    sudo install -Dm755 "pacget" "/usr/src/$name/pacget/usr/bin/pacget"
+
```
Comment (wizard-28) [inline packages/pacget-git/pacget-git.pacscript:38]:
```suggestion
	sudo install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
	sudo install -Dm755 "pacget" "/usr/src/$name/pacget/usr/bin/pacget"
```

Fix spacing & remove empty line

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**